synapse/lib/agenda.py

```diff
@@ -856,7 +856,7 @@ async def _runJob(self, user, appt):
             starttime = self._getNowTick()
             success = False
             try:
-                opts = {'user': user.iden, 'view': appt.view}
+                opts = {'user': user.iden, 'view': appt.view, 'vars': {'_iden': appt.iden}}
```

Comment:
If we dont end up with something like `$auto.iden` these should be converted `_iden` -> `croniden` ( et al )